labs/accessing-optimizely-event-data/metadata.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Accessing Optimizely Enriched Event Data
+summary: This lab contains a handy CLI for exploring and downloading Optimizely Enriched Event Data.
+revisionDate: '2020-07-20'
+labels:
+  - data
+  - analysis
+  - odata
+  - enriched
+  - events
+
+author:
+  name: Pete Koomen
+  headshot: 
+  bio: 
+  title: Co-founder
+  company: Optimizely
+
+seo:
+  title: Accessing Optimizely Enriched Event Data
+  description: This lab contains a handy CLI for exploring and downloading Optimizely Enriched Event Data.
+  ogTitle:
+  ogDescription:
+  ogImage:
+  twitterImage:
+  noindex: true
```

Comment:
I think this should be `false` so that we _do_ index. Although I'm not convinced it's actually doing anything yet

Reply:
Fixed.